interpreter/nodev8/v8.go

```diff
@@ -1877,6 +1932,26 @@ func (d *v8Data) readIntrospectionData(ef *pfelf.File, syms libpf.SymbolFinder)
 	}
 
 	// Add some defaults when needed
+	if d.version >= v8Ver(11, 9, 0) {
+		// the class hierarchy changed: HeapObject no longer
+		// derives from Object. This confuses gen-postmortem-metadata.py
```

Comment:
Neat script - maybe we should name it more prominently somewhere to help people dig into this unwinder.

Reply:
We do mention it in the main comment at the top of the file: https://github.com/open-telemetry/opentelemetry-ebpf-profiler/blob/44a0c25/interpreter/nodev8/v8.go#L28-L33

as well as various other places throughout v8.go